src/data/faq.json

```
@@ -91,6 +91,17 @@
                 "title": "News",
                 "id": "news",
                 "accordion": [
+                    {
+                        "title": "Statement regarding the Log4J vulnerability",
+                        "anchor": "log4j",
+                        "active": true,
+                        "textblock": [
+                            "Statement regarding the Log4J vulnerability.",
+                            "Log4j is a widely used Java-based utility for logging error messages in applications. The Log4j library is widely used in a very broad range of enterprise software and services from many major vendors. A vulnerability related to the Log4j library was discovered recently and allows remote code execution.",
+                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and TSI join the worldwide effort and have taken action. This includes amongst others a version update to 2.16.0",

```

Change of grammar and specific mention of server (not app).

```suggestion
                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and TSI have joined the worldwide effort and taken action. This includes amongst others a server version update to 2.16.0",
```